plugin-code/src/main/java/com/trivago/cluecumber/rendering/pages/pojos/Feature.java

```diff
@@ -43,6 +49,7 @@ public String toString() {
                 '}';
     }
 
+    // description is not taken into account in comparaison
```

Comment:
Description should be taken into account.

Reply:
just a question regarding your example project, there are a lot of json including the same Feature 2 and it is considered as the same Feature.... Why ??

Reply:
Sorry I don't get the question. Could you give me an example?

Reply:
for instance :  scenario_failure.json,  scenario_with_hook_failures.json and scenario with passing_and_and_failing all contain "Feature 2"... and are all considered as "Feature 2" in the report why ?

Reply:
Yes this is a dilemma. Because the question is if we should assume that features with the same name but different descriptions are meant to be different. If your feature is just named "Test" then the description would distinguish them. I have to think about this. But ok until then, let's leave out the description.